pkg/controller/services.go

```diff
@@ -932,6 +938,13 @@ func (cont *AciController) opflexDeviceChanged(obj apicapi.ApicObject) {
 			for node, devices := range cont.nodeOpflexDevice {
 				if node == obj.GetAttrStr("hostName") {
 					for _, device := range devices {
+						if obj.GetAttrStr("hostName") == name {
+							if device.GetDn() != obj.GetDn() && device.GetAttr("state") == "connected" {
+								cont.log.Info("Found a device that is in connected state, this happens during a live migration")
+								cont.log.Info("Restarting container")
```

Comment:
Add the name of the opflex device found in connected state here

Reply:
Although it shouldn't matter too much, since we will lost the pod logs almost instantly